src/test/java/com/scottlogic/competition_engine/application/TestControllerTests.java

```diff
@@ -0,0 +1,32 @@
+package com.scottlogic.competition_engine.application;
+
+import org.junit.Test;
```

Comment:
again, the tests for the UserService should be in its PR, this PR should just be the build stuff